controller.rst

```diff
@@ -241,10 +251,9 @@ are used for rendering templates, sending emails, querying the database and
 any other "work" you can think of. When you install a new bundle, it probably
 brings in even *more* services.
 
-When extending the base controller class, you can access any Symfony service
+When extending the base ``Controller`` class, you can access any Symfony service
```

Comment:
Do you mean we have to tell that the controller has to be configured as a service ? I updated the text to reflect that.